### DISCUSSION TITLE
Add support for sending/enqueuing message when network is down

### DISCUSSION THREAD

**Discussion by abrac** [d99kris/nchat#313] [Ideas]
When I send a message when my wifi is off or the internet is not working, the message disappears, and I am never able to get it back again. Is there a way for it to be recovered? Ideally, I would like nchat to keep it in an outbox/queue, so that it sends automatically when the network is restored, similar to how normal WhatsApp works.

**Comment by d99kris** [comment]
This is currently not supported. I'll change this discussion category to a feature request and will look into it, but it might not be a quick/simple thing to support.
